src/formatting_stack/strategies.clj

```diff
@@ -166,13 +167,15 @@
                                        ;; Temporary, until https://git.io/Jeaah is a thing:
                                        true)))) refresh-dirs)]}
   (->> files
-       (filter (speced/fn [^string? filename]
-                 (if-not (read-ns-decl filename)
-                   true
-                   (let [file (-> filename File.)]
-                     (->> refresh-dirs
-                          (some (fn [dir]
-                                  (impl/dir-contains? dir file))))))))))
+       (partitioning-pmap (speced/fn [^string? filename]
+                            (if-not (read-ns-decl filename)
+                              filename
+                              (let [file (-> filename File.)]
+                                (when (->> refresh-dirs
+                                           (some (fn [dir]
+                                                   (impl/dir-contains? dir file))))
+                                  filename)))))
+       (filter identity)))
```

Comment:
`(remove nil?)` or `(filter some?)` May be a bit more descriptive.

Reply:
`filter some?` assumes that the given coll is not a coll of booleans. That always can change over time.

Similarly `remove nil?` keeps `false` values, which creates a tight assumption about the given coll's type.

`filter identity` is unequivocal in signaling that it only cares about Clojure's _truthiness_ semantics, which is the programmer's intent here.